### PR TITLE
CONNECT-154 Add style classes as input to components

### PR DESCRIPTION
**Pull request by jihusby**
Every component now has a css class name as input. This class is assigned to the outer element of the component.
Can be used for styling of the component.

Example:
<style>
  .style-class {
    max-width: 400px;
  }
</style>
<lib-info-panel [header]="header" [content]="content" [class]="'style-class'">

### CODE REVIEW

File: projects/wonka-ui/karma.conf.js [hunk 1]
```diff
@@ -11,7 +11,6 @@ module.exports = function (config) {
       require('karma-jasmine-html-reporter'),
       require('karma-coverage-istanbul-reporter'),
       require('@angular-devkit/build-angular/plugins/karma'),
-      require('karma-spec-reporter')
```
Comment (emmaLitzler) [inline projects/wonka-ui/karma.conf.js:14]: I added karma-spec-reporter in order to have a cleaner report in the CI. It gives a list of all the tests and indicates which ones pass or not. For this to work, we need 'progress' as reporters.

Comment (jihusby) [reply projects/wonka-ui/karma.conf.js]: Ah ok. Fixed.

File: projects/wonka-ui/karma.conf.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ module.exports = function (config) {
       reports: ['html', 'lcovonly', 'text-summary'],
       fixWebpackSourcePaths: true
     },
-    reporters: ['spec', 'kjhtml'],
+    reporters: ['progress', 'kjhtml'],
```
Comment (emmaLitzler) [inline projects/wonka-ui/karma.conf.js:24]: We need "reporters: ['spec', 'kjhtml']" in order for karma-spec-reporter to work.

File: projects/application/package-lock.json [hunk 3]
```diff
@@ -14483,87 +14483,6 @@
         "chart.js": "^2.9.2",
         "ng2-charts": "^2.3.0",
         "tslib": "^1.9.0"
-      },
-      "dependencies": {
```
Comment (emmaLitzler) [inline projects/application/package-lock.json:14487]: Why is this removed?

Comment (jihusby) [reply projects/application/package-lock.json:14487]: Not removed, only moved as far as I can see. package-lock.json is auto-generated from package.json.

File: projects/wonka-ui/karma.conf.js (hunk 2 above)
Comment (jihusby) [reply projects/wonka-ui/karma.conf.js:24]: My bad. Fixed.

File: projects/application/karma.conf.js [hunk 4]
```diff
@@ -20,6 +20,7 @@ module.exports = function (config) {
       reports: ['html', 'lcovonly', 'text-summary'],
       fixWebpackSourcePaths: true
     },
+    reporters: ['spec', 'kjhtml'],
     reporters: ['progress', 'kjhtml'],
```
Comment (emmaLitzler) [inline projects/application/karma.conf.js:24]: Now reporters is defined twice. Can we delete this line?

Comment (jihusby) [reply projects/application/karma.conf.js]: Argh! Fixed.
